src/herder/TxSetUtils.cpp

```diff
@@ -0,0 +1,379 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
```

Comment:
I don't know what you did when you rebased... but you lost your local changes to the visual studio files

Reply:
```
diff --git a/Builds/VisualStudio/stellar-core.vcxproj b/Builds/VisualStudio/stellar-core.vcxproj
index 3c01bbf08..f3c8eaa9f 100644
--- a/Builds/VisualStudio/stellar-core.vcxproj
+++ b/Builds/VisualStudio/stellar-core.vcxproj
@@ -310,6 +310,7 @@ exit /b 0
     <ClCompile Include="..\..\src\herder\PendingEnvelopes.cpp" />
     <ClCompile Include="..\..\src\herder\simulation\TxSimTxSetFrame.cpp" />
     <ClCompile Include="..\..\src\herder\SurgePricingUtils.cpp" />
+    <ClCompile Include="..\..\src\herder\test\TestTxSetUtils.cpp" />
     <ClCompile Include="..\..\src\herder\TransactionQueue.cpp" />
     <ClCompile Include="..\..\src\herder\QuorumTracker.cpp" />
     <ClCompile Include="..\..\src\herder\test\HerderTests.cpp" />
@@ -319,6 +320,7 @@ exit /b 0
     <ClCompile Include="..\..\src\herder\test\UpgradesTests.cpp" />
     <ClCompile Include="..\..\src\herder\TxQueueLimiter.cpp" />
     <ClCompile Include="..\..\src\herder\TxSetFrame.cpp" />
+    <ClCompile Include="..\..\src\herder\TxSetUtils.cpp" />
     <ClCompile Include="..\..\src\herder\Upgrades.cpp" />
     <ClCompile Include="..\..\src\historywork\BatchDownloadWork.cpp" />
     <ClCompile Include="..\..\src\herder\QuorumIntersectionCheckerImpl.cpp" />
@@ -703,10 +705,12 @@ exit /b 0
     <ClInclude Include="..\..\src\herder\PendingEnvelopes.h" />
     <ClInclude Include="..\..\src\herder\simulation\TxSimTxSetFrame.h" />
     <ClInclude Include="..\..\src\herder\SurgePricingUtils.h" />
+    <ClInclude Include="..\..\src\herder\test\TestTxSetUtils.h" />
     <ClInclude Include="..\..\src\herder\TransactionQueue.h" />
     <ClInclude Include="..\..\src\herder\QuorumTracker.h" />
     <ClInclude Include="..\..\src\herder\TxQueueLimiter.h" />
     <ClInclude Include="..\..\src\herder\TxSetFrame.h" />
+    <ClInclude Include="..\..\src\herder\TxSetUtils.h" />
     <ClInclude Include="..\..\src\herder\Upgrades.h" />
     <ClInclude Include="..\..\src\historywork\BatchDownloadWork.h" />
     <ClInclude Include="..\..\src\herder\QuorumIntersectionChecker.h" />
diff --git a/Builds/VisualStudio/stellar-core.vcxproj.filters b/Builds/VisualStudio/stellar-core.vcxproj.filters
index 1e8c50c7f..c3b64393b 100644
--- a/Builds/VisualStudio/stellar-core.vcxproj.filters
+++ b/Builds/VisualStudio/stellar-core.vcxproj.filters
@@ -1233,6 +1233,12 @@
     <ClCompile Include="src\generated\rust\bridge.cpp">
       <Filter>rust\generated</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\src\herder\test\TestTxSetUtils.cpp">
+      <Filter>herder\tests</Filter>
+    </ClCompile>
+    <ClCompile Include="..\..\src\herder\TxSetUtils.cpp">
+      <Filter>herder</Filter>
+    </ClCompile>
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="..\..\lib\util\cpptoml.h">
@@ -2144,6 +2150,12 @@
     <ClInclude Include="src\generated\rust\bridge.h">
       <Filter>rust\generated</Filter>
     </ClInclude>
+    <ClInclude Include="..\..\src\herder\test\TestTxSetUtils.h">
+      <Filter>herder\tests</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\src\herder\TxSetUtils.h">
+      <Filter>herder</Filter>
+    </ClInclude>
   </ItemGroup>
   <ItemGroup>
     <None Include="..\..\AUTHORS" />
```

Reply:
I suspect it's that you had merge conflicts and just discarded your own changes. Always do a diff between your old branch and new rebased branch (that's how I found this right away).

Reply:
I cleaned up the git history and added back the Visual Studio files. 
Re: the missing file, I remember seeing the conflict on the .vcxproj files, but my the diff tool on my vscode showed no difference, so I just accepted the incoming change. Super weird. Anyways, it's fixed now and will keep an eye out in the future.